wordpress-mobile/WordPress-iOS
# Stories: Changes Kanvas editor to use white confirm + next buttons.

Moves the confirm button to the top and customizes the icons to use a white Gridicons based variant.

See https://github.com/tumblr/kanvas-ios/pull/91 for the Kanvas changes.

### Testing

* Open to a Blog Details page
* Tap the FAB
* Create a new Story Post
* Take / Select an image
* Ensure that the "next" button in the upper right is white and matches the screenshots
* Tap on the Text control in the bottom left: <img src="https://user-images.githubusercontent.com/3250/108665538-61376680-7492-11eb-9b1f-00bba8df8d70.png" width="50">

* Ensure that the "confirm" button in the upper right is white and matches the screenshots

| Before | After |
| ------ | ----- |
| <a href="https://user-images.githubusercontent.com/3250/108665413-14ec2680-7492-11eb-99ab-921ca28523df.png"><img src="https://user-images.githubusercontent.com/3250/108665413-14ec2680-7492-11eb-99ab-921ca28523df.png" width="300"></a> | <a href="https://user-images.githubusercontent.com/3250/108664425-f9801c00-748f-11eb-939f-1c8bc9a53bb2.png"><img src="https://user-images.githubusercontent.com/3250/108664425-f9801c00-748f-11eb-939f-1c8bc9a53bb2.png" width="300"></a> |

| Before | After |
| ------ | ----- |
| <a href="https://user-images.githubusercontent.com/3250/108665419-1cabcb00-7492-11eb-84f0-d3e6c934214f.png"><img src="https://user-images.githubusercontent.com/3250/108665419-1cabcb00-7492-11eb-84f0-d3e6c934214f.png" width="300"></a> | <a href="https://user-images.githubusercontent.com/3250/108664477-13b9fa00-7490-11eb-8e58-92f95e2778bb.png"><img src="https://user-images.githubusercontent.com/3250/108664477-13b9fa00-7490-11eb-8e58-92f95e2778bb.png" width="300"></a> |


PR submission checklist:
